examples/03_connectivity/plot_sphere_based_connectome.py

```diff
@@ -0,0 +1,175 @@
+# Computing within spheres averaged time-series
+# ---------------------------------------------
+
+###############################################################################
+# We can compute the mean signal within **spheres** of a fixed radius around
+# a sequence of (x, y, z) coordinates with the object
+# :class:`nilearn.input_data.NiftiSpheresMasker`.
+# So we collect the regions coordinates in a numpy array
+import numpy as np
+
+coords = np.vstack((
+    power.rois['x'],
+    power.rois['y'],
+    power.rois['z'],
+)).T
+
+print('Stacked power coordinates in array of shape {0}.'.format(coords.shape))
+
+###############################################################################
+# and define spheres masker, with small enough radius to avoid regions overlap.
+from nilearn import  input_data
+
+spheres_masker = input_data.NiftiSpheresMasker(
+    seeds=coords, smoothing_fwhm=4, radius=5.,
+    detrend=True, standardize=True, low_pass=0.1, high_pass=0.01, t_r=2.5)
+
+###############################################################################
+# Voxel-wise time-series within each sphere are averaged. The resulting signal
+# is then prepared by the masker object: Detrended, cleaned from counfounds,
+# band-pass filtered and **standardized to 1 variance**.
+timeseries = spheres_masker.fit_transform(fmri_filename,
+                                          confounds=confounds_filename)
+
+###############################################################################
+# Correlation matrix computation
+# ------------------------------
+
+###############################################################################
+# All starts with the estimation of the signals **covariance**. Here the number
+# of ROIs exceeds the number of samples,
+print('time series has {0} samples'.format(timeseries.shape[0]))
+
+###############################################################################
+# in which situation the graphical lasso **sparse inverse covariance**
+# estimator captures well the covariance **structure**.
+from sklearn.covariance import GraphLassoCV
+
+covariance_estimator = GraphLassoCV(verbose=1)
+
+###############################################################################
+# We just fit our regions signals into the `GraphLassoCV` object
+covariance_estimator.fit(timeseries)
+
+###############################################################################
+# and get the ROI-to-ROI covariance matrix.
+matrix = covariance_estimator.covariance_
+print('Covariance matrix has shape {0}.'.format(matrix.shape))
+
+###############################################################################
+# Now remember that our signals have been **standardized** by the masker. So we
+# already have our pairwise correlations available with no further effort.
+
+# check diagonal is 1.
+print('Covariance matrix has diagonal {0}.'.format(
+    np.unique(np.diagonal(matrix))))
```

Comment:
Maybe you don't have to write code for this (more code is more stuff to understand for the beginner user), and simple comment on the figure showing the matrix.